scalastyle-config.xml

```diff
@@ -0,0 +1,79 @@
+<scalastyle>
+  <name>Scalastyle standard configuration</name>
+
+  <check level="error" enabled="true" class="org.scalastyle.file.FileTabChecker"/>
+  <check level="error" enabled="true" class="org.scalastyle.file.WhitespaceEndOfLineChecker"/>
+  <check level="error" enabled="true" class="org.scalastyle.file.NewLineAtEofChecker"/>
+
+  <!--Indentation checker, whilst useful gives false positives on the file headers, so disabled for now-->
+  <check level="error" enabled="false" class="org.scalastyle.file.IndentationChecker"/>
+
+  <check level="error" enabled="true" class="org.scalastyle.file.FileLineLengthChecker">
+    <parameters>
+      <parameter name="maxLineLength"><![CDATA[120]]></parameter>
+    </parameters>
+  </check>
+
+  <check level="error" enabled="true" class="org.scalastyle.scalariform.RedundantIfChecker"/>
+  <check level="error" enabled="true" class="org.scalastyle.scalariform.SimplifyBooleanExpressionChecker"/>
+
+  <check level="error" enabled="true" class="org.scalastyle.scalariform.UppercaseLChecker"/>
+
+  <check level="error" enabled="true" class="org.scalastyle.scalariform.ClassNamesChecker">
+    <parameters>
+      <parameter name="regex"><![CDATA[[A-Z][A-Za-z]*]]></parameter>
```

Comment:
That was the default suggestion for the config. Since it didn't trip against anything, I let it. If we need to we can relax it later.